navigation/create_control/launch/create_control.launch

```diff
@@ -0,0 +1,20 @@
+<launch>
+  <arg name="robot_id"  default="$(optenv ID 1)"      doc="Unique identifier of the robot [1-Inf.)"/>
+  <arg name="ns" value="create$(arg robot_id)"/>
+
+  <!-- Load controller configuration from YAML file to parameter server -->
+  <rosparam file="$(find create_control)/config/create_2_control.yaml" command="load"/>
+
+  <!-- need to be set separatedly, since they depend on the robot id, and that cannot be parametrized
+  in the yaml file.-->
+  <param name="/create1/mobile_base_controller/odom_frame_id" value="/create$(arg robot_id)_tf/odom"/>
+  <param name="/create1/mobile_base_controller/base_frame_id" value="/create$(arg robot_id)_tf/base_footprint"/>
```

Comment:
I don't think you will need a namespace in the parameter because the whole launch file is already there.

```suggestion
  <param name="mobile_base_controller/odom_frame_id" value="$(arg tf_prefix)/odom"/>
  <param name="mobile_base_controller/base_frame_id" value="$(arg tf_prefix)/base_footprint"/>
```

Reply:
Done